Resources/Prototypes/_LostParadise/Traits/languages.yml

```
@@ -11,8 +11,18 @@
     inverted: false
     species:
     - Human
-  - !type:CharacterItemGroupRequirement
-    group: TraitsLanguagesBasic
+  - !type:CharacterTraitRequirement
+    inverted: true
+    traits:
+    - DwarfLanguage
```

yamllint

[error] wrong indentation: expected at least 5

(indentation)